src/AutoComplete/AutoComplete.js

```diff
@@ -497,7 +497,7 @@ class AutoComplete extends Component {
         />
         <Popover
           {...popoverProps}
-          style={styles.popover}
+          style={prepareStyles(Object.assign(styles.popover, (popoverProps || {}).style))}
```

Comment:
@mivanichok_twitter is having some close concern with the overriding power of `popoverProps`.
It should definitely have the higher priority so people can change whatever they want.
That should be more better like this.
```jsx
const {
  style: popoverStyle,
  .. popoverOther,
} = popoverProps;

//...

<Popover
  style={Object.assign((), styles.popover, popoverStyle)}
  canAutoPosition={false}
  anchorOrigin={anchorOrigin}
  targetOrigin={targetOrigin}
  open={open}
  anchorEl={anchorEl}
  useLayerForClickAway={false}
  onRequestClose={this.handleRequestClose}
  animated={animated}
  animation={animation}
  {...popoverOther}
>
```